projects/kserve/visualizations/kserve-llm/store/lts_parser.py

```diff
@@ -106,15 +116,19 @@ def generate_lts_metadata(results, import_settings):
 
 def generate_lts_results(results):
     results_lts = types.SimpleNamespace()
-    # Throughout value (scalar, in token/ms)
 
+    # Throughout value (scalar, in token/ms)
     results_lts.throughput = _generate_throughput(results)
 
     # Time Per Output Token value
     results_lts.time_per_output_token = _generate_time_per_output_token(results)
 
-    # Time To First Token values for all of the calls (vector)
-    results_lts.time_to_first_token = _generate_time_to_first_token(results)
+    if _is_streaming(results):
+        # Inter Token Latency (ms)
+        results_lts.inter_token_latency = _generate_inter_token_latency(results)
+
+        # Time To First Token values for all of the calls (vector)
+        results_lts.time_to_first_token = _generate_time_to_first_token(results)
```

Comment:
can you make these fields `None` if not streaming?
should `Streaming` be exposed as a label? 🤔